roles/sro_deploy_art_bundle/files/builder_buildconfig.yml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  labels:
+    app: ci-artifacts
+  name: ci-artifacts-builder
+  namespace: ci-artifacts
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: ci-artifacts-builder:latest
+      namespace: ci-artifacts
+  source:
+    type: Dockerfile
+    dockerfile: |
+      FROM registry.fedoraproject.org/fedora:latest
+
+      RUN dnf -y update \
+          && dnf -y install podman chkconfig java-headless gcc jq git
+
+      RUN curl -LO https://golang.org/dl/go1.17.linux-amd64.tar.gz \
+          && tar -C /usr/local -xzf go1.17.linux-amd64.tar.gz
+
+      RUN export PATH=$PATH:/usr/local/go/bin \
+          && dnf install -y git make \
+          && git clone https://github.com/operator-framework/operator-registry \
+          && cd operator-registry \
+          && make bin/opm \
+          && mv bin/opm /usr/local/go/bin/
```

Comment:
@pacevedom, any reason for building `opm` here, instead of [downloading](https://github.com/operator-framework/operator-registry/releases/download/v1.19.5/linux-amd64-opm) it? (as @omertuc noticed)